.prettierignore

```diff
@@ -7,8 +7,6 @@ node/
 
 .git
 
-.yarnrc.yml
```

Comment:
No longer automatically generated, so we can adhere to Prettier formatting rules now.